Chaincase.UI/Pages/SendWhoPage.razor

```diff
@@ -12,18 +16,21 @@
 <ion-content class="ion-padding">
     <h1>Sending to who?</h1>
 
-    <ion-label>Address</ion-label>
+    <ion-label>Address or <code>bitcoin:</code> uri</ion-label>
     @* TODO put a paste button at the end of the input*@
-    <IonInput @bind-Value="AddressString" placeholder="Enter address..."/>
+    <IonInput @bind-Value="ViewModel.DestinationString" placeholder="Enter address..."/>
 
     @*qr code reader goes here*@
-    <IonButton OnClick="OnContinue" expand="block">NEXT</IonButton>
+    <IonButton OnClick="OnContinue" expand="block" disabled="@(!IsAddressValid)">NEXT</IonButton>
 </ion-content>
 
 @code {
 
     private string Label { get; set; }
-    private string AddressString { get; set; }
+
+    private string _addressString;
+
+    private bool IsAddressValid => ViewModel.Address is not null;
```

Comment:
will eventually implement address validation? I would imagine the wasabi standard library prob has some relevant functions

Reply:
yeah it already has that. not null implies it is an NBitcoin`BitcoinAddress` and has been validated